Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -284,3 +319,52 @@ private func fetchProject(projectOrParam: Either<Project, Param>, shouldPrefix:
 
   return projectProducer
 }
+
+private func reward(from backing: Backing?, inProject project: Project) -> Reward {
+  return backing?.reward
+    ?? project.rewards.filter { $0.id == backing?.rewardId }.first
+    ?? Reward.noReward
+}
+
+private func canShowRewardsScreen(
+  _: Project,
+  _: RefTag?,
+  state: PledgeStateCTAType?
+) -> Bool {
+  guard let state = state else {
+    return false
+  }
+  return userCanSeeNativeCheckout() && (state == .pledge || state == .viewRewards)
+}
+
+private func canShowManageViewPledgeScreen(
```

Comment:
After we remove the unused params, these function names should fit on one line making it a bit more readable IMO.

```swift
private func canShowRewardsScreen(state: PledgeStateCTAType?) -> Bool { .. }
private func canShowManageViewPledgeScreen(_ project: Project, state: PledgeStateCTAType?) -> Bool { .. }
private func shouldGoToDeprecatedViewBacking(_ project: Project, state: PledgeStateCTAType?) -> Bool { .. }
private func shouldGoToDeprecatedManagePledge(_ project: Project, state: PledgeStateCTAType?) -> Bool { .. }
```